src/main/java/io/anserini/document/TweetDocument.java

```diff
@@ -16,18 +16,16 @@
 
```

Comment:
How about just rename to `Tweet`? See below.

Reply:
I think we should keep this unchanged since its name is consistent with other document classes e.g. `TrecDocument`, `TrecwebDocument`.

Reply:
okay, sg